testapp/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from solo.models import SingletonModel
+
+
+class Configuration(SingletonModel):
+    """
+    configuration for export/import admin color theme
+    """
+
+    theme = models.ForeignKey("Theme", on_delete=models.CASCADE, related_name="theme")
+
+    def __str__(self) -> str:
+        return self.theme.name
+
+
+class Theme(models.Model):
+    """object to change color theme in admin"""
+
+    name = models.CharField(_("theme_name"), max_length=120)
+    primary = models.CharField(_("primary color"), max_length=60)
+    secondary = models.CharField(_("secondary color"), max_length=60)
+    accent = models.CharField(_("accent color"), max_length=60)
+    primary_fg = models.CharField(_("primary_fg"), max_length=60)
```

Comment:
Can you use actually usefull labels? Repeating the attribute name is not usefull. At least remove the ``_``

Reply:
Done